site/src/components/message.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https://www.figma.com/proto/amgDNGgGYy2v2Unc6Eet46/Q%26A-Private-Messages?page-id=0%3A1&type=design&node-id=337-533145&viewport=-15969%2C-17033%2C0.75&scaling=min-zoom&starting-point-node-id=337%3A548136&show-proto-sidebar=1 allowfullscreen
+classType: Component
+classPrefixes:
+  - message
+---
+
+# Message
+
+Use message component for chat or messaging communications.
+
+## Usage
+
+Here is an example of a standard usage of message component.
+
+```html
+<article class="message">
+	<p class="message-time">May 29, 2023, 4:50PM</p>
+	<div class="persona message-persona">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="Avatar for Jane Doe" />
+		</figure>
+	</div>
+	<div class="message-content">
+		<div class="persona message-content-header">
+			<div class="persona-details">
+				<p class="persona-name font-weight-semibold">Q&A Moderator</p>
+				<p class="font-size-xs">Moderator</p>
+			</div>
+			<details class="message-content-options">
```

Comment:
Popover misses the `popover-content` part, either add it here with the correct popover classes or replace it with a button fir cimplicity. Same for all examples including the pattern/chat.

```html
<details class="popover>
    <summary class="button">:</summary>
    <div class="popover-content">Popover content.</div>
</details>
```

Reply:
updated